detekt-baseline-main.xml

```diff
@@ -930,6 +930,10 @@
     <ID>ReturnCount:ExceptionHandling.kt$ExceptionHandling$override fun toProblem(throwable: Throwable, status: StatusType): ThrowableProblem?</ID>
     <ID>ReturnCount:OffenderService.kt$OffenderService$fun getInfoForPerson(crn: String, deliusUsername: String, ignoreLao: Boolean): PersonInfoResult</ID>
     <ID>ReturnCount:TaskService.kt$TaskService$fun reallocateTask(requestUser: UserEntity, taskType: TaskType, userToAllocateToId: UUID, id: UUID): AuthorisableActionResult&lt;ValidatableActionResult&lt;Reallocation&gt;&gt;</ID>
+    <ID>ReturnCount:StatusUpdateService.kt$StatusUpdateService$fun create( applicationId: UUID,
+      statusUpdate: Cas2ApplicationStatusUpdate,
```

Comment:
I don't know if this is best practice, but couldn't see a way of having less returns.

Reply:
also not sure if the ignore should be in `detekt-baseline-main` or `detekt-baseline`?